README.old.md

```diff
@@ -0,0 +1,2 @@
+# hello-react-front-end
+This projects aims to set react as a separate app from rails
```

Comment:
- Please remove this readme from your repo to make your repo look more professional 👍 👍